low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -195,14 +201,18 @@ def _raid_health_monitor(self):
                             data = fs.read().rstrip()
                         if self.FAULT_RESOLVED in data:
                             self.alert_type = self.FAULT
-                            self._alert_msg = "RAID disks present in %s RAID array, needs synchronization." %device
+                            self._alert_msg = "RAID disks present in %s RAID array, needs synchronization." %device +\
+                                " If fault persists for more than 1 or 2 days, Please contact Seagate support."
```

Comment:
just change this to
 If fault persists for more than 2 days, Please contact Seagate support

Reply:
done.